services/libs/tinybird/pipes/leaderboards_codebase_size.pipe

```diff
@@ -0,0 +1,42 @@
+DESCRIPTION >
+    Leaderboard ranking projects by codebase size, measured as the net lines of code changed
+    (total insertions minus deletions) across all git commits. Only includes projects with
+    git platform integrations.
+
+NODE leaderboards_codebase_size_projects
+DESCRIPTION >
+    Retrieves all projects that have git integration enabled
+
+SQL >
+    SELECT id, name, slug, segmentId, logoUrl
+    FROM insights_projects_populated_ds
+    WHERE has(connectedPlatforms, 'git')
+    GROUP BY id, name, slug, segmentId, logoUrl
+
+NODE leaderboards_codebase_size_activities
+DESCRIPTION >
+    Calculates net lines changed per project by summing git insertions and subtracting deletions
+
+SQL >
+    SELECT segmentId, SUM(gitInsertions) - SUM(gitDeletions) AS linesChanged
```

Comment:
this looks not right -  Lines changed should be gitInsertions + gitDeletions. And we already have this in `activityRelations_deduplicated_cleaned_ds`.`gitChangedLines`